datafusion/core/src/execution/session_state.rs

```diff
@@ -554,7 +554,7 @@ impl SessionState {
             );
         }
         let statement = statements.pop_front().ok_or_else(|| {
-            DataFusionError::NotImplemented(
+            DataFusionError::Plan(
```

Comment:
```suggestion
            plan_datafusion_err!(
```
to support backtracing